Adjust Checkpoint input style

wcole3 opened the pull request:
Slight adjustments to the Checkpoint dropdown in response to #1586 .

I would appreciate feedback from UX/UI folks, I think that these changes allow the filtering input to not overlap the dropdown arrow, but the right-align might be too much.

Current Filtering
![image](https://github.com/user-attachments/assets/42cfddeb-f285-49b4-a6b4-3bbab8a909e4)

Updated Filtering (See Below)


Current Long Checkpoint Listing
![image](https://github.com/user-attachments/assets/0f7a0bc1-7e59-4c06-b0f9-27073adfdcb5)

Updated Long Checkpoint Listing (See below)

Current Short Checkpoint Listing
![image](https://github.com/user-attachments/assets/9da1bf8b-2790-44e9-9ee0-492689da7484)

Updated Short Checkpoint Listing (See below)


Additionally, not sure what the best way to address the ul.options formatting, I think adjusting them to right aligned would address the issue, but unsure if that is desired.

MNeMoNiCuZ commented:
Can you show a version where you expand the menu, and it shows a too long path? Does it show the whole thing? Or is this what you weren't able to figure out yet?

---

### Feedback:
I feel like the right-alignment is noone's friend. It just adds too much noise to one area and it's harder to parse.

wcole3 commented:
That's what I couldn't figure out
![image](https://github.com/user-attachments/assets/9e2e93fa-7caf-4dc3-b584-618ff3beaf21)

wcole3 commented:
Looking at it a bit more, I think you're right that the right align is not great so I removed it.  Also fixed an issue causing the input area to only be a fraction of the total area.

Updated LaF
![image](https://github.com/user-attachments/assets/74389282-ed94-4793-96b7-c5f52aa0f4cb)
![image](https://github.com/user-attachments/assets/6f020806-eed5-461e-a5c7-2e4e38508470)

The behavior of the dropdown for longer paths is still very wonky, I could get it to overflow, but I couldn't find a way to make the size match the size of the longest item. 
![image](https://github.com/user-attachments/assets/a566da1f-9e25-4a09-a12c-14e8ffcfff18)
![image](https://github.com/user-attachments/assets/2c788a0f-3b04-4656-a21a-ae12fdb57656)

I did not add the above css changes, and I'll wait to see if others have ideas or if the scenario should be addressed another way.

MNeMoNiCuZ commented:
Nice!
Left-aligned is an improvement, and even the text going outside is an improvement in readability even if not perfect.

wcole3 commented:
So I was able to find the right syntax to cap the length of the window at the max element.  I added that to the PR, but if its not desired I can back it out.

![image](https://github.com/user-attachments/assets/4cb4bd1b-181e-470f-8c47-54c850bf300c)

![image](https://github.com/user-attachments/assets/665cbf52-95fc-4c63-b72d-5315427ffe57)

wcole3 commented:
Can also cap the max width so that the popup doesn't go too crazy:

![image](https://github.com/user-attachments/assets/55326c3d-f671-4584-8535-dffa902d0d6a)

MNeMoNiCuZ commented:
I prefer the maximum length one myself but I'm open to other preferences.
Maybe it should be limited to the total width of the page, so it doesn't go to second monitor or outside of view.

Good job with this quick fix!
